README.md

```diff
@@ -0,0 +1,136 @@
+# backplane-tools
+
+backplane-tools offers an easy solution to install, remove, and upgrade a useful set of tools for interacting with OpenShift clusters, managed or otherwise.
+
+## Table of Contents
+<!-- toc -->
+- [FAQ - How do I...](#faq-how-do-i)
+  - [List available tools](#list-available-tools)
+  - [Install everything](#install-everything)
+  - [Install a specific thing](#install-a-specific-thing)
+  - [Upgrade everything](#upgrade-everything)
+  - [Upgrade a specific thing](#upgrade-a-specific-thing)
+  - [Remove everything](#remove-everything)
+  - [Remove a specific thing](#remove-a-specific-thing)
+- [Design](#design)
+  - [Directory Structure](#directory-structure)
+  - [Installing](#installing)
+  - [Upgrading](#upgrading)
+  - [Removing](#removing)
+<!-- tocstop -->
+
+## FAQ - How do I...
+Quick reference guide
+
+### List available tools
+```shell
+backplane-tools install --help
```

Comment:
Would it make sense to have an up-to-date list here so one can see what's in it for them before installing the tool?